app/base-env/Dockerfile

```diff
@@ -70,20 +70,30 @@ RUN /llvm.sh ${CLANG_VERSION} && /gcc.sh ${GCC_VERSION}
 ENV PATH="/usr/lib/llvm-${CLANG_VERSION}/bin/:$PATH"
 
 # Newer CMake
+FROM dependencies as cmake
```

Comment:
What is the advantage of starting a new stage here ?

Reply:
None for now. I was thinking about what we can parallelise with buildkit later on. The steps after cmake could all start together. But I did not implement it. I can remove it